content/en/docs/tasks/configure-pod-container/assign-cpu-resource.md

```diff
@@ -148,6 +148,50 @@ Delete your Pod:
 kubectl delete pod cpu-demo --namespace=cpu-example
 ```
 
+## How CPU requests and limits work under the hood
+
+### On Linux systems
+
```

Comment:
Please wrap the long "lines" appropriately.
See https://sembr.org for the preferred line wrapping style.